pyproject.toml

```diff
@@ -66,3 +66,6 @@ default-tag = "0.0.0"
 
 [tool.versioningit.write]
 file = "src/konnektor/_version.py"
+
+[tool.pytest.ini_options]
+addopts = "--ignore=dev --cov=konnektor"
```

Comment:
I wouldn't do coverage by default if you can add it - coverage has a really big overhead on pytest (up to 400% in some cases).

You definitely don't want to run it locally while you're developing things.

Reply:
Good point, moved it out of defaults.